modules/core/src/main/java/org/apache/ignite/internal/util/ByteUtils.java

```diff
@@ -74,6 +86,17 @@ public static byte[] longToBytes(long l) {
         return putLongToBytes(l, new byte[8], 0);
     }
 
+    /**
```

Comment:
What about ints?

Reply:
No one needs it right now

Reply:
So, someone else will have some fun while debugging similar issue in a year with ints, right? ;)

Reply:
Maybe

Reply:
Well, I strongly suggest to add comparable counterparts to int.